### PR TITLE
SSCS SYA web chat test

### PR DESCRIPTION
**Pull request by divisathyan**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/SSCS-9604

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Review by srochani — APPROVED** [review]

File: apps/sscs/demo/base/kustomization.yaml [hunk 1]
```diff
@@ -3,4 +3,5 @@ kind: Kustomization
 resources:
   - ../../base
 patchesStrategicMerge:
-  - ../../sscs-core-frontend/demo.yaml
\ No newline at end of file
+  - ../../sscs-core-frontend/demo.yaml
+  - ../../sscs-tribunals-frontend/demo.yaml
```
Comment (timja) [inline apps/sscs/demo/base/kustomization.yaml:7]: this app appears to be not kustomized in demo so this isn't needed...

File: apps/sscs/sscs-tribunals-frontend/demo.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
```
[the remark this answers is not in the document]
Comment (divisathyan) [reply apps/sscs/sscs-tribunals-frontend/demo.yaml]: Removed

File: apps/sscs/demo/base/kustomization.yaml (hunk 1 above)
Comment (divisathyan) [reply apps/sscs/demo/base/kustomization.yaml]: Removed

File: k8s/demo/common/sscs/sscs-tribunals-frontend.yaml [hunk 3]
```diff
@@ -17,7 +17,7 @@ spec:
       replicas: 2
       useInterpodAntiAffinity: true
       ingressClass: traefik-no-proxy
-      image: hmctspublic.azurecr.io/sscs/tribunals-frontend:prod-5a5d799-20211012142811 #{"$imagepolicy": "flux-system:sscs-tribunals-frontend"}
+      image: hmctspublic.azurecr.io/sscs/tribunals-frontend:pr-1245-f40accf-20211012133236 #{"$imagepolicy": "flux-system:sscs-tribunals-frontend"}
```
[the remark this answers is not in the document]
Comment (divisathyan) [reply k8s/demo/common/sscs/sscs-tribunals-frontend.yaml]: Followed the documentation and created demo-sscs-tribunals-frontend. Added the demo-image-policy to apps/sscs/automation/kustomization.yaml

**Review by timja — APPROVED** [review]

**Review by srochani — APPROVED** [review]

**Pull request merged**